Source/Core/Core/IOS/Network/NCD/Manage.h

```diff
@@ -33,6 +35,12 @@ class NetNCDManage : public Device
     IOCTLV_NCD_GETWIRELESSMACADDRESS = 0x8,  // NCDGetWirelessMacAddress
   };
 
+  struct Context
```

Comment:
Maybe a dumb question - do you expect more values to be added here eventually?  Is that why there's a context for a single value?

Reply:
Yes, there will be more values in this struct, just like in NCD.